collection-scripts/gather_sos

```diff
@@ -38,7 +38,7 @@ else
 fi
 
 # Default to some plugins if SOS_ONLY_PLUGINS is not set
-SOS_ONLY_PLUGINS="${SOS_ONLY_PLUGINS-block,cifs,crio,devicemapper,devices,iscsi,lvm2,memory,multipath,nfs,nis,nvme,podman,process,processor,selinux,scsi,udev}"
+SOS_ONLY_PLUGINS="${SOS_ONLY_PLUGINS-block,cifs,crio,devicemapper,devices,iscsi,lvm2,memory,multipath,nfs,nis,nvme,podman,process,processor,selinux,scsi,udev,openstack_edpm}"
```

Comment:
the plugins have not been updated to account for the fact that we are now logging to the journal

as a result we should consider if we need to enable the generic logs plugin by default to collect the combined journal log or if the podman plugin will collect the logs form all conatienrs.

Reply:
this is the logs plugin 
https://github.com/sosreport/sos/blob/main/sos/report/plugins/logs.py
the alternative is to use the podman plugin with the logs option

https://github.com/sosreport/sos/blob/main/sos/report/plugins/podman.py#L118-L121

however that will not collect the libvirt logs.

we could do both but either option wont filter passwords or other info that is logged so we have to keep that in mind.

Reply:
we can do that in follow up.
Having the basic logging (journal etc. on edpm) is covered, as testing shows